features/wijzigingen.feature

```diff
@@ -0,0 +1,100 @@
+        Gegeven gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999993653" en geen einddatum
+        En op 12 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999993653"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" een item met de waarde "999993653"
+
+    Scenario: Gebruiker heeft een volgindicatie op een persoon en er worden meerdere wijziging doorgevoerd in de registratie bij de persoon
+        Gegeven gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999993653" en geen einddatum
+        En op 10 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999993653"
+        En op 12 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999993653"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" exact één (1) item met de waarde "999993653"
+
+    Scenario: Gebruiker heeft een volgindicatie op een persoon en er is geen wijziging doorgevoerd in de registratie bij de persoon
+        Gegeven gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999993483" en geen einddatum
+        En sinds 5 maart 2021 is er in de BRP registratie geen wijziging doorgevoerd in de gegevens van de persoon "999993483"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999993483"
+
+    Scenario: Wijzigingen vóór het zetten van de volgindicatie worden niet gemeld
+        Gegeven op 4 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999990457"
+        En gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999990457" en geen einddatum
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999990457"
+
+    Scenario: Gebruiker zet een volgindicatie met een einddatum in de toekomst
+        Gegeven gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992855" met einddatum "2021-03-16"
+        En op 12 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999992855"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" een item met de waarde "999992855"
+
+    Scenario: Gebruiker zet een volgindicatie met een einddatum in het verleden of vandaag
```

Comment:
Scenario titel klopt niet. De gebruiker zet geen volgindicatie met een einddatum in het verleden. Op het moment dat de gebruiker de volgindicatie zet, ligt het einddatum in de toekomst. Ik zou eerder zeggen dat de volgindicatie van de gebruiker is verlopen